docs/source/defaults.rst

```diff
@@ -0,0 +1,49 @@
+:orphan:
+
+.. toggle::
+
+    .. testcode:: default, 1, 2, 3, 4, 5
```

Comment:
@DevPranjal what does `default, 1, 2, 3, 4, 5` ?

Reply:
In the doctests we have (in some places) grouped the tests like `1, 2, 3` so that variables from one doctest are no used in the other. Here `1, 2, 3` implies that this testcode will defined for each of those groups (and separately). Not putting these will define this setup only for the `default` group. 

There is also an alternative to this. The docs say we can put `*` instead of naming all the groups, but that does not seem to work when I tried it out.

Reply:
@DevPranjal let's move with that and if @sdesrozis have another idea we can make a follow-up PR

Reply:
Great!